build/includes/release.mk

```diff
@@ -96,5 +96,5 @@ do-release: $(ensure-build-image)
 	@echo "Now go make the $(RELEASE_VERSION) release on Github!"
 
 build-release:
-	cd $(agones_path) && gcloud builds submit . --substitutions _VERSION=$(base_version) --config=./build/release/cloudbuild.yaml $(ARGS)
-
+	docker run --rm $(common_mounts) -w $(workdir_path)/build/release/ $(build_tag) \
+		gcloud builds submit . --substitutions _VERSION=$(base_version) $(ARGS)
```

Comment:
I can't tell from the commit message why the `--config` option is no longer being passed into cloud build. Was that intentional?

Reply:
It's not necessary. The default is the `cloudbuild.yaml` found in the working folder.

Reply:
Actually - that is a really good point. We should continue to upload the source for Agones, since we will need to build everything eventually.

Will fix!